monkey/tests/unit_tests/infection_monkey/network_scanning/test_http_fingerprinter.py

```diff
@@ -9,14 +9,14 @@
 from infection_monkey.i_puppet import PortScanData
 from infection_monkey.network_scanning.http_fingerprinter import HTTPFingerprinter
 
-OPTIONS = {"http_ports": [80, 443, 1080, 8080, 9200]}
+OPTIONS = {"http_ports": [80, 444, 1080, 8080, 9200]}
```

Comment:
Based on the commit message, we should change 80 to something else too to avoid unexpected behavior if we ever make any changes to the tests.